package.json

```diff
@@ -33,7 +33,7 @@
     "middleware"
   ],
   "dependencies": {
-    "@hapi/joi": "17.x.x",
+    "joi": "17.x.x",
     "@types/express": "4.x.x",
     "@types/hapi__joi": "16.x.x"
```

Comment:
Shouldn't this change to `@types/joi`? We may need to wait until it gets in sync with the last release of `joi`

Reply:
Yup, should probably jump "@types/hapi__joi` to `@types/joi`. But as you mentioned it still hasn't caught up, so that's probably another, later, Pull Request.

This PR should be mergeable now, and would remove the deprecation warning at npm/yarn install time.

Reply:
Many thanks

Reply:
Re: `@types/joi`, there is a PR to update the types DefinitelyTyped/DefinitelyTyped#46241 but it looks like the progress has stalled.